packages/blitz-next/src/error-boundary.test.tsx

```diff
@@ -426,7 +427,7 @@ test("supports reset via resetKeys right after error is triggered on component m
 test("should support not only function as FallbackComponent", () => {
   const consoleError = console.error as MockedFunction<(args: unknown[]) => void>
 
-  const FancyFallback = forwardRef(({error}: ErrorFallbackProps) => (
+  const FancyFallback = React.forwardRef(({error}: ErrorFallbackProps) => (
```

Comment:
why these changes?

Reply:
I had problem passing these tests, forwardRef was not defined. Adding the React indicator fixed that problème.